### ISSUE TITLE
Tutorial unclear or out of date

### ISSUE THREAD

**Issue by NicholasHattabaugh** [Urigo/Ionic2-MeteorCLI-WhatsApp#32]
I've tried several times to get this tutorial to work, i'm new to angular2 and meteor but would love to use it for my latest project, i can get this to build for the browser but nothing appears, the build fails for ios,  and I have a mile long list of errors in the web console when it does run, all of the other tutorials on the meteor website I was able to get to work, if anyone could help me it would be greatly appreciated.

**Comment by NicholasHattabaugh** [comment]
these are the three errors I get in the browser console, 
modules.js?hash=a269f0f…:31333 ERROR Error: No provider for ApplicationInitStatus!

Unhandled Promise rejection: No provider for ApplicationInitStatus! ; Zone: <root> ; Task: Promise.then ; Value: Error: No provider for ApplicationInitStatus!

modules.js?hash=a269f0f…:1285 Error: Uncaught (in promise): Error: No provider for ApplicationInitStatus!
Error

**Comment by NicholasHattabaugh** [comment]
this is after completing the bootstrapping section of the tutorial using the meteor cli

**Comment by JayDijkstra** [comment]
Also a problem understanding this tutorial completely. Getting the same errors and now have a weird SASS compile error:

> /client/main.scss: Scss compiler error: unclosed parenthesis

I just want to go as easy trough this tutorial as the old Socially tutorial, but every time something goes wrong!

**Comment by JayDijkstra** [comment]
The SASS Compiler error was an ; in my markup language, dumb me :).
I fixed my No Provider for ApplicationInitStatus by adding the BrowserModule to my app.module.ts file.


`import { BrowserModule } from '@angular/platform-browser';`

Dont forget to add it to your Imports declarations as well (inside of your NgModule)

```
@NgModule({
  declarations: [
    MyApp,
    ChatsPage
  ],
  imports: [
    IonicModule.forRoot(MyApp),
    MomentModule,
    BrowserModule
  ],
})
```
Hope this will solve your problem as well!

**Comment by trancaohuy** [comment]
I also get the same error when running through the bootstrapping tutorial. So far, the only thing that is working for me is using the webpack version of the tutorial. However, it seems like the webpack version doesn't support offline database. Does anyone know if the meteor tutorial support offline database?

**Comment by ghost** [comment]
Is anyone able to make the tutorial work with meteor CLI? I am getting a bunch of errors and its driving me crazy. It seems the tutorial is really out of date.

**Comment by jessonjoy** [comment]
yeah I too am facing errors on the first steps of this tutorials. I copied the code verbatim and I get the following errors:

imports/app/app.module.ts (8, 11): Argument of type '{ declarations: typeof MyApp[]; imports: typeof ErrorHandler[]; entryComponents: typeof MyApp[]; ...' is not assignable to parameter of type 'NgModule'.

**Comment by jessonjoy** [comment]
I added a custom error hander module and class and imported them and it works now.

```
import {GlobalErrorHandler} from "./error.module";
import {GlobalErrorModule } from "./error.module";

@NgModule({
    declarations: [ MyApp ],
    imports: [BrowserModule, GlobalErrorModule],
    bootstrap: [ MyApp ],
    providers: [
        { provide: GlobalErrorHandler, useClass: GlobalErrorHandler}
    ]
})
```

**Comment by sam17896** [comment]
I just want a simple boiler-plate meteor application with angular2.. after following step 1 my application is started at port 3000 but nothing is displaying.. after sometimes i get "aww snap" error.
